arangodb-net-standard/DocumentApi/PatchDocumentQuery.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+
+namespace ArangoDBNetStandard.DocumentApi
+{
+    public class PatchDocumentQuery
+    {
+        public bool? KeepNull { get; set; }
+
+        public bool? MergeObjects { get; set; }
+
+        public bool? WaitForSync { get; set; }
+
+        public bool? IgnoreRevs { get; set; }
+
+        public bool? ReturnOld { get; set; }
+
+        public bool? ReturnNew { get; set; }
+
+        public bool? Silent { get; set; }
```

Comment:
I'm interested to know what happens to the response if we set Silent to true, could you test that?